src/view/actions/sendEvent.jsx

```diff
@@ -66,6 +66,11 @@ const xdmFieldDescription = (
 
 const wrapGetInitialValues = getInitialValues => ({ initInfo }) => {
   const { personalization = {}, ...otherSettings } = initInfo.settings || {};
+  if (personalization.defaultPersonalizationEnabled === true) {
+    personalization.defaultPersonalizationEnabled = "true";
+  } else if (personalization.defaultPersonalizationEnabled === false) {
+    personalization.defaultPersonalizationEnabled = "false";
+  }
```

Comment:
nass: we can use ternary 
`personalization.defaultPersonalizationEnabled = personalization.defaultPersonalizationEnabled ? "true" : "false";
`

Reply:
This wouldn't work because when defaultPersonalizationEnabled is undefined we want to keep it as undefined.